specification/programmableconnectivity/Azure.ProgrammableConnectivity/apis/network.tsp

```diff
@@ -0,0 +1,58 @@
+import "./common.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace Azure.ProgrammableConnectivity;
+
+/// Interfaces
+
+interface Networks {
```

Comment:
No, it means that we are going to allow developers to either make two calls or one call. I put some thoughts about this in the doc here:specification/programmableconnectivity/Azure.ProgrammableConnectivity/examples/LocationInterface_Verify_MaximumSet_Gen.json. If a developer wants to make multiple calls about one device they could get lower latency by using the network API and then using the network code I don't know how significant this will end up being though, so we can always decide to depricate.

Reply:
Right, yes, sorry, I forgot that the conclusion of that discussion was to keep the Network API as well as supporting passing in IP addresses as a network identifier.

For anyone else reading this, I think this doc is what you meant to link to: https://microsoft.sharepoint.com/:w:/r/teams/CommunicationsSDK/Shared%20Documents/Alpaca%20-%20(Gateway)%20-%20General/API%20and%20SDK%20Review%20Processes/[NetworkAPIProposal.docx](https://microsoft.sharepoint.com/:w:/r/teams/CommunicationsSDK/Shared%20Documents/Alpaca%20-%20(Gateway)%20-%20General/API%20and%20SDK%20Review%20Processes/NetworkAPIProposal.docx?d=wd4bcfc631f054cc1a4676e79795e0599&csf=1&web=1&e=n4kecS)?d=wd4bcfc631f054cc1a4676e79795e0599&csf=1&web=1&e=n4kecS